python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -164,8 +201,12 @@ def __init__(self, era, algo = 'csvv2', sfFileName = None, verbose = 0):
         self.branchNames_central_and_systs = {}
         for central_or_syst in self.central_and_systs:
             if central_or_syst == "central":
+                if era.find("FastSim"):
+                    self.branchNames_central_and_systs[central_or_syst] = "Jet_btagSF_FS"
                 self.branchNames_central_and_systs[central_or_syst] = "Jet_btagSF"
             else:
+                if era.find("FastSim"):
+                    self.branchNames_central_and_systs[central_or_syst] = "Jet_btagSF_FS_%s" % central_or_syst
                 self.branchNames_central_and_systs[central_or_syst] = "Jet_btagSF_%s" % central_or_syst
```

Comment:
If I understand correctly, this module will be called twice in the new postporcess. btagSFProducer(args.era+"FastSim", algo="deepcsv"), and btagSFProducer(args.era, algo="deepcsv"). Then this line should within the else condiction of  "if era.find("FastSim")", no?